mx.jruby/suite.py

```diff
@@ -143,21 +222,10 @@ def mavenLib(mavenDep, sha1, sourceSha1, license):
             "license": "EPL-1.0",
         },
 
-        # Depends on jruby-core extracting jni libs in lib/jni
-        "jruby-lib-jni": {
```

Comment:
I thought the native *.so needed to be real files on the filesystem, is this not required?